packages/devops/scripts/deployment/dumpE2eDataToS3.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Run directly from a nightly cron job, so needs full setup of path etc.
+
+# Set PATH to include depencencies
+export PATH=/home/ubuntu/.local/bin:/home/ubuntu/.yarn/bin:/home/ubuntu/.config/yarn/global/node_modules/.bin:/home/ubuntu/.nvm/versions/node/v12.18.3/bin:/usr/local/bin:$PATH
+
+DIR=$(dirname "$0")
+export STAGE=$(${DIR}/../utility/getEC2TagValue.sh Stage)
+
+# Set the branch based on STAGE
+if [[ "$STAGE" == "e2e" ]]; then
+  echo "Dumping E2E data from E2E baseline instance to S3"
+  mkdir -p dumps
+
+  echo "Dumping survey_response"
+  psql -U tupaia tupaia -c "\copy survey_response to 'dumps/survey_response.csv' delimiter ',' csv header"
+
+  echo "Dumping answer"
+  psql -U tupaia tupaia -c "\copy answer to 'dumps/answer.csv' delimiter ',' csv header"
```

Comment:
Perhaps we should also add `dhis_sync_queue`, `dhis_sync_log`?

Reply:
Yeah great point! I did think about doing this, but then wondered if there were other tables (like `ms1_sync_queue` etc.) and decided it might get out of hand for no real benefit. It will result in a somewhat corrupted db state, but only tables that won't be used for e2e at this stage, and keeps the process simple. What do you think?

Reply:
I agree with the overall approach! Just for those tables in specific, there may be data sync leftovers in prod from last night - I guess we wouldn't want both `dev` and `e2e` trying to sync those

Reply:
No but we actually already have something in place to prevent that:
```
      // don't sync changes that happened before the server was started, to avoid multiple
      // deployments syncing identical changes that came from the overnight snapshot
      criteria.change_time = {
        comparator: '>',
        comparisonValue: SERVER_START_TIME,
      };
```

Reply:
Great!!!